.gitignore

```diff
@@ -5,3 +5,5 @@ log
 *.pid
 npm-debug.log
 config/index.js
+.vscode
+cordlr.json
```

Comment:
Where is it suppose to be? I will change it to correct folder and maybe add a **sample.cordlr.json** file?